featuretools/computational_backends/pandas_backend.py

```diff
@@ -267,17 +310,18 @@ def _feature_type_handler(self, f):
         else:
             raise UnknownFeature(u"{} feature unknown".format(f.__class__))
 
-    def _calculate_identity_features(self, features, entity_frames):
-        entity_id = features[0].entity.id
-        return entity_frames[entity_id][[f.get_name() for f in features]]
+    def _calculate_identity_features(self, features, df_trie):
```

Comment:
Wouldn't there be extra columns if other features were already calculated?

Reply:
i see. im not sure if it is possible for there to be any other features already calculated before we calculate identity features. 

that being said, there probably isn't value in adding the check I propose. if necessary columns is working properly, it shouldn't matter. 

going a step further, if necessary columns is working, im not sure the assert we already is needed either, but it would catch errors, so I believe it's worth keeping

Reply:
I think there could be precalculated features?

Agreed about the existing assert.